vite.config.js

```diff
@@ -58,6 +58,14 @@ export default defineConfig(({ mode }) => {
   return {
     server: {
       port: 3000,
+      fs: {
+        allow: [
+          // search up for workspace root
+          searchForWorkspaceRoot(process.cwd()),
+          // Allow fs access to local build of wasm
+          "../matrix-rust-sdk-crypto-wasm/pkg",
```

Comment:
Every developer's file system and exact setup of linked packages could look different - for instance, I experience this same problem when I use a linked `../matrix-js-sdk` package. I think the following, resolving the symlinks with `realpath`, would work for us both:

```suggestion
          // The crypto WASM module is imported dynamically. Since it's common
          // for developers to use a linked copy of matrix-js-sdk or Rust
          // crypto (which could reside anywhere on their file system), Vite
          // needs to be told to recognize it as a legitimate file access.
          await realpath(
            "node_modules/matrix-js-sdk/node_modules/@matrix-org/matrix-sdk-crypto-wasm",
          ),
```

Reply:
A complication: that path might not exist if not using a linked copy of matrix-js-sdk. (because the dependency will instead be hoisted to node_modules/@matrix-org/matrix-sdk-crypto-wasm)